docs/docs/get-started/setup-a-dbt-cloud-project.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 3
+sidebar_label: Setup a dbt cloud project
+---
+
+# Setup Lightdash to connect to your dbt cloud project
+
+In this tutorial, you'll setup Lightdash and connect it to your dbt project on [dbt cloud](https://cloud.getdbt.com).
+dbt cloud provides a development environment that makes it really easy to your develop your dbt project code, with 
+Lightdash you can also visualise your dbt models while developing you code.
+
+**Prerequisites**
+ - A dbt cloud account. If you don't have one, [sign up for a free here](https://cloud.getdbt.com/signup/)
+ - Your dbt cloud account should have at least one environment (data warehouse) configured
+ - Your dbt cloud account environment should be using dbt version `0.20.0` or higher
+
+## 1. Get your `accountId` and `projectId`
+
+Login to [dbt cloud](https://cloud.getdbt.com) and follow these instructions to get your account and project id:
+
+1. Make sure you have the correct project selected in the drop down settings
+2. Get your `accountId` from the URL after `/accounts/`
+3. Get your `projectId` from the URL after `/projects/`
+
+![screenshot](assets/dbt-cloud-account-project.png)
+
+## 2. Get your `environmentId`
+
+Use the sidebar to see all your environments. To connect to your dbt IDE you must select your development credentials with
+type `type: development`. This should be the environment you usually use when developing in the dbt cloud IDE.
+
+![screenshot](assets/dbt-cloud-sidebar.png)
+
+![screenshot](assets/dbt-cloud-env-select.png)
+
+Once you've located your environment follow these steps to get your environment id:
+
+1. Get your `environmentId` from the URL after `/environments/`
+2. Check that your environment is using dbt `0.20.0` or above (you can change this in environment settings)
+
+![screenshot](assets/dbt-cloud-env-details.png)
+
+## 3. Get your `apiKey`
+
+You can get your personal api key by visiting your [API Access - Your profile](https://cloud.getdbt.com/#/profile/api/).
+
+:::info
+
+**It's a secret!** Remember to keep your api key safe as it enables access to your dbt cloud account.
+
+:::
+
+## 4. Create your lightdash.yml file
+
+Use `accountId`, `projectId`, `environmentId`, and `apiKey` from the previous steps and create a file called `lightdash.yml`
+
+```yaml
+# lightdash.yml
+version: '1.0'
+
+projects:
+  - name: 'default'
+    type: 'dbt_cloud_ide'
+    accountId: # your id
+    projectId: # your id
+    environmentId: # your id
+    apiKey: # your api key
```

Comment:
Amend example
```suggestion
    account_id: # your id
    project_id: # your id
    environment_id: # your id
    api_key: # your api key
```
After amending this locally I also got the parsing error `ParseError: Lightdash config file successfully loaded but invalid: Field at /projects/0/account_id must be string` because the values are by default numbers. So I had to wrap them in single quotes. Should we mention that in the example or change or validation to also support numbers ?

Reply:
Not sure what to do here, the ids are numbers. But always feels weird to make ids an actual number haha.

What would you prefer? Accepting numbers and strings doesn't seem too bad

Reply:
String already means people can type any invalid id in there that doesn't look like a number

Reply:
Oh one thing to add: dbt cloud returns a different response in these cases:
1.  if you give a valid id but doesn't exist like '999999' - you get a meaningful response
2. you give a weird id like 'aaa11happy22no' - you get a garbage response

Reply:
> dbt cloud returns a different response
We should open a bug on their repo. I don't see the reason why they are treated differently.

> Accepting numbers and strings doesn't seem too bad
Yeah, I think this is the best approach

Reply:
> should open a bug on their repo

I think this is proprietary (i.e. not in dbt core)

> Yeah, I think this is the best approach

on it!